pwa/src/services/getStatusColorAndIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { faCheck, faTriangleExclamation, faXmark } from "@fortawesome/free-solid-svg-icons";
+
+export const getStatusColor = (value: any): any => {
```

Comment:
I'd expect a `getStatusColor` function to return a `TStatusColor` and not `any`.